src/ui/components/fiber-tree/ScrollSelectedIntoViewIfNeeded.tsx

```diff
@@ -1,16 +1,18 @@
 import * as React from "react";
-import { useSelectedId } from "../../utils/selection";
+import { useSelectedId, useHighlightedId } from "../../utils/selection";
 import { getBoundingRect, getOverflowParent } from "../../utils/layout";
 import { useTreeViewSettingsContext } from "./contexts";
 
 export const ScrollSelectedIntoViewIfNeeded = () => {
```

Comment:
I think now it should be renamed into `ScrollFiberIntoViewIfNeeded`

Reply:
Fixed